react/src/screens/BookDetail/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component,Fragment } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import BookDetailCommentaries from '../../components/BookDetailCommentaries';
+import BookDetailData from '../../components/BookDetailData';
+import BookDetailSuggestions from '../../components/BookDetailSuggestions';
+import {data} from '../../mock.js';
+import routes from '../../constants/routes';
+
+import './styles.css';
+
+class BookDetail extends Component {
+  constructor(props) {
```

Comment:
I'd write `state = { error: null, errorInfo: null }` in order to avoid writing a `constructor`